tree/ntuple/v7/doc/specifications.md

```diff
@@ -649,14 +649,24 @@ The child fileds are named `_0` and `_1`.
 #### std::tuple<T1, T2, ..., Tn>
 
 A tuple is stored using an empty mother field with $n$ subfields of type `T1`, `T2`, ..., `Tn`. All types must have RNTuple I/O support.
-The child fileds are named `_0`, `_1`, ...
+The child fields are named `_0`, `_1`, ...
 
 #### std::bitset<N>
 
 A bitset is stored as a repetitive leaf field with an attached `Bit` column.
 The bitset size `N` is stored as repetition parameter in the field meta-data.
 Within the repetition blocks, bits are stored in little-endian order, i.e. the least significant bits come first.
 
+#### std::unique_ptr<T>, std::optional<T>
+
+A unique pointer and an optional type have the same on disk representation.
```

Comment:
It seems 'implicit' for the rest but is the type fixed also in the `unique_ptr` case *(i.e. no support for polymorphism?)

Reply:
That's right, there is no polymorphism support for `T`. Clarified in the specs.